docs/reference/query-dsl/span-queries.asciidoc

```diff
@@ -18,7 +18,7 @@ other span queries.
 
 Wraps a <<query-dsl-term-query,`term`>>, <<query-dsl-range-query,`range`>>,
 <<query-dsl-prefix-query,`prefix`>>, <<query-dsl-wildcard-query,`wildcard`>>,
-<<query-dsl-regexp-query,`regexp`>>, or <<query-dsl-fuzzy-query,`fuzzy`>> query.
+or <<query-dsl-regexp-query,`regexp`>> query.
 
```

Comment:
There is no alternative for span query with fuzzy terms if we completely remove the fuzzy query builder. I guess that's ok because it's a corner case but yep we're losing this. Bye bye fuzzy query !